integreat_cms/cms/views/pages/page_actions.py

```diff
@@ -299,6 +303,10 @@ def cancel_translation_process_ajax(
         page_translation.all_versions.update(currently_in_translation=False)
     # Get new (respectively old) translation state
     translation_state = page.get_translation_state(language_slug)
+    messages.success(
+        request,
+        f"Cancelled translation process for page {page} and language {page_translation.language}",
```

Comment:
I still see two issues here:

1. This string also appears in English in the German version of the system. I think this should be how to fix it. 

```suggestion
        _("Cancelled translation process for page {page} and language {page_translation.language}"),
```
Then by running `./tools/translate.sh` it should generate a new translation file where you can then insert the German translation for this sentence.

2. The success messages are created in bulk and therefore overpopulating the screen (s. screenshot).
For that we wrote the `iter_to_string` function that puts the list into a string and then just shows that one message. You can find an example of this in use [here](https://github.com/digitalfabrik/integreat-cms/blob/develop/integreat_cms/cms/views/bulk_action_views.py#L179)

![grafik](https://github.com/user-attachments/assets/2e6f5370-15b5-4fe4-98c3-37f3997f4200)